test/wasmex_test.exs

```diff
@@ -108,18 +108,21 @@ defmodule WasmexTest do
       Wasmex.Memory.write_binary(memory, index, string)
 
       assert {:ok, [104]} ==
-              Wasmex.call_function(instance, "string_first_byte", [index, String.length(string)])
+               Wasmex.call_function(instance, "string_first_byte", [index, String.length(string)])
     end
   end
 
   describe "when instantiating with imports" do
-    test "call_function: using_imported_function_add_number(u32) -> (u32) function" do
-      imports = %{"env" => %{
-        "imported_sum" => {[:uint8, :uint8], [:uint8], &(&1 + &2)}
-      }}
+    test "call_function: using_imported_sum3(u32, u32, u32) -> (u32) function" do
+      imports = %{
+        "env" => %{
+          "imported_sum3" => {[:i32, :i32, :i32], [:i32], &(&1 + &2 + &3)}
+        }
+      }
```

Comment:
Hmm, could we keep both sum functions (`imported_sum` and `imported_sum3`)?

I know that you implemented a working function signature for calling callbacks, but technically you wouldn't need to with just this test. Having two functions with different signatures forces our implementation to properly handle (and not hardcoded) signatures.

You could optionally implement any other function that seems to be a good example (`multiply` maybe) so this makes more sense than just two sum-functions.

Or, maybe even better, you could keep the `imported_sum3` functions, and add another that sums float (f32) values. This would help us test that we support different type.